src/build/arg/arg_value.rs

```diff
@@ -0,0 +1,77 @@
+use std::fmt::Display;
+
+/// TODO
+#[derive(Debug, Default, Clone)]
+pub struct ArgValue<'help> {
+    pub(crate) value: &'help str,
+    pub(crate) about: Option<&'help str>,
+    pub(crate) hidden: bool,
```

Comment:
With the reflection API, do these need to be `pub(crate)`?

Reply:
The reflection API is generally only used for the `clap_generate` library and they are added as needed. While all the members here are available in the reflection API, anything we add in the future might now and I would like it if we consistently add `pub(crate)` to all of them.

Reply:
imo we should work to remove `pub(crate)` from struct members as much as possible.
- It shows a weakness in our reflection
- It breaks encapsulation, making it harder to know who does what

Reply:
IIRC we tried to do that. We realised that it's quite a lot of effort for low reward.